src/main/java/racing/domain/RacingInfo.java

```diff
@@ -0,0 +1,36 @@
+package racing.domain;
+
+import java.util.List;
+
+public class RacingInfo {
+
+    private final List<Car> cars;
+    private final Round round;
+    private final MoveRule moveRule;
+
+    public RacingInfo(List<Car> cars, Round round, MoveRule moveRule) {
+        this.cars = cars;
+        this.round = round;
+        this.moveRule = moveRule;
+    }
+
+    public static RacingInfo init(String carNames, int roundNumber) {
+        final List<Car> cars = CarGenerator.createCars(carNames);
+        MoveRule moveRule = initMoveRule(4);
+        Round round = new Round(roundNumber);
+
+        return new RacingInfo(cars, round, moveRule);
+    }
```

Comment:
```suggestion
    private RacingInfo(List<Car> cars, Round round) {
        this.cars = cars;
        this.round = round;
    }

    public static RacingInfo of(String carNames, int roundNumber) {
        final List<Car> cars = CarGenerator.createCars(carNames);
        Round round = new Round(roundNumber);
        return new RacingInfo(cars, round);
    }
```

`MoveRule`은 `CarGenerator`에서 생성해 사용하고 있어 `RacingInfo`에서는 불필요한 것 같습니다. 
정적 팩토리 메서드사용 한 것이라면, 네이밍 컨벤션을 맞춰 보는 것은 어떨까요?
참고 - [정적 팩토리 메서드는 왜 사용할까?](https://velog.io/@ljinsk3/%EC%A0%95%EC%A0%81-%ED%8C%A9%ED%86%A0%EB%A6%AC-%EB%A9%94%EC%84%9C%EB%93%9C%EB%8A%94-%EC%99%9C-%EC%82%AC%EC%9A%A9%ED%95%A0%EA%B9%8C)

Reply:
헉 오늘 처음 읽기 시작한 책의 첫 장이 정적 팩토리 메소드 내용이었는데 제 코드에 바로 관련 내용이 있었네요!!ㅎㅎ
네이밍 컨벤션 맞춰서 수정했습니다!